Made Branch available as parameter

phifogg opened the pull request:
Addressing Issue #2 

Branch is now an attribute which can be used from TypeScript / YAML file.

chiarnglin commented:
@phifogg 
hey daniel! firstly, thanks a bunch for contributing! keep up with the open source spirit! 
i haven't had a chance to dig into your commits in detail yet, but noted there's a LOT of node modules you are committing as a part of this PR. can you find a way to fix that?

phifogg commented:
Changes applied to code:
- moved branch from body to header when calling ServiceNow API
- removed auto-branch selection as it tried to select a branch with name 'merge' when run in a PR
- exposed branch as attribute to be used in YAML file

phifogg commented:
Example YAML content:

      - name: ServiceNow CI/CD Apply Changes
        uses: phifogg/sncicd-apply-changes@1.2.0
        env:
          snowUsername: ${{ secrets.SNOW_USERNAME }}
          snowPassword: ${{ secrets.SNOW_PASSWORD }}
          snowSourceInstance: ${{ secrets.SNOW_TEST_INSTANCE }}
          appSysID: ${{ secrets.APP_SYS_ID }}
          branch: 'instances/dev'
